app/models/stream.rb

```diff
@@ -119,7 +119,7 @@ def self.build_or_update!(data = {})
     stream.set_bounding_box(latitude, longitude) unless stream.has_bounds?
     stream.save!
 
-    MeasurementsCreator.call(stream, measurements_attributes)
+    MeasurementsCreator.new.call(stream, measurements_attributes)
```

Comment:
this and line 109 seem very similar, any reason why it's called measurements_attributes here and just measurements on 109 - it's a bit confusing, I was trying to figure out what's the difference, cause the name doesn't really say much